src/components/rangeUsePlanPage/index.js

```diff
@@ -72,13 +72,42 @@ const Base = ({
   const getPlanId = () =>
     match.params.planId || location.pathname.charAt('/range-use-plan/'.length)
 
+  // for plans extending past agreement date, extend usage
+  const appendUsage = (plan, Plan) => {
```

Comment:
I don't think this second Plan argument is used.
```suggestion
  const appendUsage = plan => {
```

Reply:
I would also recommend moving this function outside of the component and into a utils file, so it can be re-used in other parts of the application if necessary.

Reply:
these are fixed